src/domain/occurrence/CreateOccurrencePage.tsx

```diff
@@ -179,4 +229,33 @@ const CreateOccurrencePage: React.FC = () => {
   );
 };
 
+const useInitialFormValues = () => {
+  const query = useQuery();
+
+  // initial pre-filled values from event wizard step 1
+  const initialDate = query.get('date');
+  const initialStartsAt = query.get('startsAt');
+  const initialEndsAt = query.get('endsAt');
+
+  const initialFormValues = React.useMemo(() => {
+    if (initialDate && initialEndsAt && initialStartsAt) {
+      const valuesAreValid =
+        isValidDate(new Date(initialDate)) &&
+        isValidTime(initialStartsAt) &&
+        isValidTime(initialEndsAt);
+      return valuesAreValid
+        ? {
+            ...defaultInitialValues,
+            date: new Date(initialDate),
+            startsAt: initialStartsAt,
+            endsAt: initialEndsAt,
+          }
+        : defaultInitialValues;
+    }
+    return defaultInitialValues;
+  }, [initialDate, initialEndsAt, initialStartsAt]);
+
+  return initialFormValues;
+};
+
```

Comment:
wasn't quite sure what would be the cleanest way to handle these initial values... ended up with this :D

Reply:
did some changes to this already for the next step